src/main/resources/io/jenkins/plugins/appcenter/releasenotes/ManualReleaseNotes/help-releaseNotes.html

```diff
@@ -0,0 +1,4 @@
+<div>
+    Release notes to be shown to users on App Center.<br/>
+    You can include build variables here like <code>${BUILD_NUMBER}</code> and they'll be expanded at build time.
```

Comment:
Are you _sure_ they'll be expanded? I have yet to include support for build variables.